.github/workflows/mako.yml

```diff
@@ -0,0 +1,37 @@
+name: mako-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+    mako-testsuite-1_1_4:
+    runs-on: ubuntu-latest
+    env:
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: mako
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: sqlalchemy/mako
+          ref: 1.1.4
+          path: mako
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```

Comment:
The version of python doesn't matter, we could do 3.9, but 3.8 is stable enough and perfect for now.